influxdb-meta/README-short.txt

```diff
@@ -0,0 +1 @@
+InfluxDB Enterprise is a highly available time series database.
```

Comment:
InfluxDB Enterprise is the commercial offering of InfluxDB, with clustering and high availability.

99 characters, BOOM!